main.go

```diff
@@ -129,11 +129,23 @@ func loadingIndicator(done chan bool) {
 	}
 }
 
-func getUserInput(prompt string) string {
+func getUserInput(prompt string, allowEmpty ...bool) string {
 	fmt.Print(prompt)
 	scanner := bufio.NewScanner(os.Stdin)
-	scanner.Scan()
-	return scanner.Text()
+
+	enforceNonEmpty := false // Default to false (meaning empty input is NOT allowed)
+	if len(allowEmpty) > 0 {
+		enforceNonEmpty = allowEmpty[0]
+	}
+
+	for {
+		scanner.Scan()
+		input := scanner.Text()
+		if input != "" || enforceNonEmpty {
```

Comment:
you should be able to just check allowEmpty directly here:

```suggestion
		if input != "" || allowEmpty {
		    return input
		}
```

Reply:
I couldn't get that to work believe it or not